app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -25,8 +25,13 @@ if [ -f /etc/os-release ] ; then
   . /etc/os-release
 fi
 
+SSL_CA_CERT=$(mktemp --suffix .crt)
```

Comment:
I think `--suffix` is a GNU extension and won't work on BSDs.

I think it's also good to clean up after yourself and do a `rm $SSL_CA_CERT` in the end.

Reply:
Are all our templates BSD compatible? I know some systems are picky about the suffix, but `curl` probably does not care. If that's the case, let's drop that.

Reply:
I think we did our best and I tried to spot all things I could. If curl doesn't care, there's not really a benefit to using it now.